.bazelci/presubmit.yml

```diff
@@ -78,14 +78,6 @@ tasks:
     working_directory: examples
     test_targets:
       - //...
-  docs_linux:
```

Comment:
Docs should still be built in CI to ensure there are no issues generating them

Reply:
done.